autosklearn/pipeline/components/data_preprocessing/balancing/balancing.py

```diff
@@ -130,3 +130,6 @@ def get_hyperparameter_search_space(dataset_properties: Optional[DATASET_PROPERT
         cs = ConfigurationSpace()
         cs.add_hyperparameter(strategy)
         return cs
+
+    def to_sklearn(self) -> Optional[BaseEstimator]:
+        return None
```

Comment:
I'm guessing there's a way to wrap this functionality into a model but that's not essential for now?

Reply:
Yes, one could basically add a hyperparameter to each model to decide whether it should do balancing or not. This component goes a different route and provides one global hyperparameter for all models that support balancing.

Reply:
Maybe this is also suboptimal because not all classifiers support re-balancing the data as of now.

Reply:
I think so, too.